src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have `margin-right`, which is fine for inline elements, but ensure that vertical margins are applied consistently either to top or bottom to avoid margin collapse issues.